### DISCUSSION TITLE
Patch API 404 - The requested resource wasn't found.

### DISCUSSION THREAD

**Discussion by shyakadavis** [pocketbase/pocketbase#4232] [Q&A] [answered]
Hi;

Wasn't sure if I was supposed to open an issue, but this felt more appropriate.

I noticed something when trying to update a record and wanted to ask for some help/guidance.

Say I have a `projects` collection, where each record has a `user` field, which is in relation to the `users` collection.

The API rule for updating is as such:

```
@request.auth.id = user
```

Now, the problem, I found, is that when the record's relation is `multiple` and not `single` in nature, it causes that API rule to break in such as way:

```shell
{
  url: 'http://localhost:8090/api/collections/projects/records/jgql0om7acb0ner',
  status: 404,
  response: {
    code: 404,
    message: "The requested resource wasn't found.",
    data: {}
  },
  isAbort: false,
  originalError: {
    url: 'http://localhost:8090/api/collections/projects/records/jgql0om7acb0ner',
    status: 404,
    data: {
      code: 404,
      message: "The requested resource wasn't found.",
      data: {}
    }
  }
}
```
```text
sql: no rows in result set
```

The logic here, for my use case at least, is that a project can have multiple collaborators, and each should have the right to edit/update the project.

My question is, how can I modify the API rule to `loop` over each user is the field, and if it finds one, permits access.

Thanks.

**Comment by ganigeorgiev** [comment, marked as answer]
You can use the any/at-least-one-of operator (`?=`) and combine it with dot-notation to target the id of the relation record. Something like:

```js
@request.auth.id != "" &&
@request.auth.id ?= user.id
```
